storage/badger/headers.go

```diff
@@ -90,6 +107,11 @@ func NewHeaders(collector module.CacheMetrics, db *badger.DB) *Headers {
 			withStore(storeChunkID),
 			withRetrieve(retrieveChunkID),
 			withResource(metrics.ResourceFinalizedHeight)),
+		timeStampCache: newCache(collector,
+			withLimit(4*flow.DefaultTransactionExpiry),
+			withStore(storeTimeStamp),
+			withRetrieve(retrieveTimeStamp),
+			withResource(metrics.ResourceFinalizedHeight)),
```

Comment:
not sure what the `WithResource` should be. Should I define a new one?

Reply:
yep should add a new one